README.md

```diff
@@ -208,6 +208,51 @@ And last but not least:
 
 Now you should be able to open `http://localhost:8000`.
 
+How to get it running with Fig
+---------------------------------
+
+You need to install [Docker](https://docs.docker.com/installation/#installation)
+and [Fig](http://www.fig.sh/install.html) in your machine and then you can
+build the image, but first you have to modify your `airmozilla/settings/local.py`
+file so it can connect to the database and memcached.
+
+```
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.postgresql_psycopg2',
+        'NAME': 'postgres',
+        'USER': 'postgres',
+        'HOST': 'db',
+        'PORT': '',
+    }
+}
+
+CACHES = {
+    'default': {
+        'BACKEND': 'django.core.cache.backends.memcached.MemcachedCache',
+        'KEY_PREFIX': 'airmoz',
+        'TIMEOUT': 6 * 60 * 60,
+        'LOCATION': 'memcached:11211'
+    }
+}
+```
+
+After this you can run the app by doing:
+
+```
+fig up
+```
+
+With this if your are on Linux you can connect to `localhost:8000` or to
+`192.168.59.103:8000` if you are on Windows or Mac.
+Due the fact that on Windows and Mac docker uses boot2docker this ip can change
+at some point, to know the exact ip address you can execute `boot2docker ip`.
+
+If you want to run all tests you can execute:
+
+```
+fig run web ./manage.py test
+```
```

Comment:
Can you add a note about how you also start the `runserver`. You know, in case you want to start it with some different options. 
Also, as another example perhaps show how you get access to the shell. E.g. `fig run web ./manage.py shell`